### PR TITLE
Modify ammo recipe and add a new recipe that uses tin

### PR DESCRIPTION
**Pull request by LoneWolfHT**
5 steel is a bit too much IMO, using tin may not be realistic but it gives tin some value.

### CODE REVIEW

**Comment by AKryukov92** [comment]
It will make them uncraftable on "the bridge" and "nostalgia" maps.

**Comment by LoneWolfHT** [comment]
> It will make them uncraftable on "the bridge" and "nostalgia" maps.

Oh really :thinking: 
What if I added a recipe with 3 steel and 2 coal then?

**Comment by LoneWolfHT** [comment]
> What if I added a recipe with 3 steel and 2 coal then?

Done

File: mods/ctf_crafting/init.lua [hunk 1]
```diff
@@ -159,7 +159,15 @@ crafting.register_recipe({
...
 	level  = 1,
 })
+
+crafting.register_recipe({
+	type   = "inv",
+	output = "shooter:ammo 1",
+	items  = { "default:steel_ingot 3", "default:coal_lump 2" },
+	always_known = true,
+	level  = 1,
+})
```
Comment (ClobberXD) [inline mods/ctf_crafting/init.lua]: Please add a newline at the end of the file.

**Comment by LoneWolfHT** [comment]
Huh, didn't seem to work. Will do tomorrow when I can get back on my computer

**Comment by ClobberXD** [comment]
It worked. The Travis CI build passed too. The newline is just not displayed in the Github diff.

**Comment by ClobberXD** [comment]
Maybe `shooter:ammo 2` as output instead of 1 for both the recipes?

**Comment by AKryukov92** [comment]
I'd rather be careful and add underpowered recipes rather than overpowered. Thats why I support 3+2=1 recipe.

**Pull request merged**